content/Documentation Guidelines - Sebastián Caraballo.md

```diff
@@ -0,0 +1,112 @@
+
+Later on in the project, the client hired a content manager who was tasked with uploading new episodes and exclusive content to the platform. As a result, the team started developing documentation for the content manager to understand all the logic behind the administration site, for example,  how to upload new content, schedule episode releases, and so on. These documents were linked on the administration's dashboard site itself for easy access.
+
+### **Documentation Tools**
+
+Here are some of the tools we use everyday to document project deliverables.
+
+#### **Oracle Apiary**
+[Oracle Apiary](https://apiary.io/) is an API design stack we use to describe all endpoints using `API Blueprint`, a really cool description language based on Markdown. It allows for specifying actions, declaring objects, and data structures. The online editor provides an automatic validation which helps developers to keep documents consistent. Here is a brief syntax example that is taken from our open-source [Rails API Template](https://github.com/rootstrap/rails_api_base):
+
+```apib
+FORMAT: 1A
+HOST: http://rails5-api-base.herokuapp.com
+
+# API BASE
+
+Rails Api Base is a boilerplate project for JSON RESTful APIs.
+
+## Users Collection [/api/v1/users]
+
+### Sign Up [POST]
+
++ Request (application/json)
+    + Attributes
+      + user (Profile Edition, required)
+
+
++ Response 200 (application/json)
+    + Headers
+
+            access-token: sO2bm_Bpdyoo8r78jZ-fqg
+            client: QADgNCWRJj0LyRruqzYbBg
+            expiry: 1489009792
+            uid: test@test.com
+
+
+    + Attributes
+      + user (Profile Response, required)
+
++ Response 401
+
+
+# Data Structures
+
+## Profile Response (object)
++ id: 100 (number)
++ email: test@test.com (string)
++ first_name: John (string)
++ last_name: Doe (string)
+
+## Profile Edition (object)
++ email: test@test.com (string)
++ password: 12345678 (string)
++ first_name: John (string)
++ last_name: Doe (string)
+```
+
+---
+
+#### **Postman**
+
+![image](images/postman.jpg)
+
+[Postman](https://www.postman.com/) is an amazing API client we use locally or with our testing environments so everyone on the team is able to inspect endpoint responses and debug. This is specially useful for frontend developers since it enables them to quickly get responses based on the request attributes. Furthermore, the environment can be easily switched in order to make a request (e.g. `staging`, `production`, and so on). Postman helps the team to better understand how the API works.
```

Comment:
```suggestion
[Postman](https://www.postman.com/) is an amazing API client we use locally or with our testing environments so everyone on the team is able to inspect endpoint responses and debug. This is especially useful for frontend developers since it enables them to quickly get responses based on the request attributes. Furthermore, the environment can be easily switched in order to make a request (e.g. `staging`, `production`, and so on). Postman helps the team to better understand how the API works.
```